Core/tests/Unit/Report/GAEStandardMetadataProviderTest.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * Copyright 2018 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Core\Tests\Unit\Report;
+
+use Google\Cloud\Core\Report\GAEStandardMetadataProvider;
+use PHPUnit\Framework\TestCase;
+
+/**
+ * @group core
+ */
+class GAEStandardMetadataProviderTest extends TestCase
+{
+    private $envs = [
+        'GAE_SERVICE' => 'my-service',
+        'GAE_VERSION' => 'my-version',
+        'GOOGLE_CLOUD_PROJECT' => 'my-project',
+        'HTTP_X_CLOUD_TRACE_CONTEXT' => 'my-traceId'
+    ];
+
+    /**
+     * @dataProvider testData
+     */
+    public function testGAEStandardMetadataProvider(
+        $envs,
+        $monitoredResource,
+        $projectId,
+        $serviceId,
+        $versionId,
+        $labels
+    )
```

Comment:
Putting the opening bracket on the same line as the parenthesis should get the CI to pass :) - `) {`

Reply:
Done!